### ISSUE TITLE
VNC enabled container cannot start when env var `SE_OPTS` contains whitespace.

### ISSUE THREAD

**Issue by vimagick** [SeleniumHQ/docker-selenium#335]
I'm try to run `selenium/node-chrome-debug` with this `docker-compose.yml`:

```yaml
chrome:
  image: selenium/node-chrome-debug
  ports:
    - "5555:5555"
    - "5900:5900"
  environment:
    - JAVA_OPTS=-Xmx512m
    - NODE_MAX_INSTANCES=2
    - NODE_MAX_SESSION=2
    - SE_OPTS=-host 1.2.3.4 -port 5555
    - HUB_PORT_4444_TCP_ADDR=5.6.7.8
    - HUB_PORT_4444_TCP_PORT=4444
  restart: always
```

I'm try to connect this node (1.2.3.4:5555) to remote hub (5.6.7.8:4444).

```bash
$ docker-compose up -d
Creating selenium_chrome_1

$ docker-compose logs -f
Attaching to selenium_chrome_1
chrome_1  | appending selenium options: -host 1.2.3.4 -port 5555
chrome_1  | Waiting xvfb...
chrome_1  | -bash: 1.2.3.4: command not found
chrome_1  | Waiting xvfb...
chrome_1  | Waiting xvfb...
chrome_1  | Waiting xvfb...

$ docker-compose ps
      Name                  Command             State                          Ports
---------------------------------------------------------------------------------------------------------
selenium_chrome_1   /opt/bin/entry_point.sh   Restarting   0.0.0.0:5555->5555/tcp, 0.0.0.0:5900->5900/tcp
```

I found that [entry_point.sh][1] cannot handle white-spaces inside `$SE_OPTS` very well.

[1]: https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeChromeDebug/entry_point.sh#L38-L49

**Comment by FelikZ** [comment]
This is related to this merge #249 , I bet.

**Comment by ddavison** [comment]
out of curiosity, instead of specifying an array, have you also tried just an object?

```yaml
...

environment:
    JAVA_OPTS: '-Xmx512m'
    NODE_MAX_INSTANCES: 2
    NODE_MAX_SESSION: 2
    SE_OPTS: '-host 1.2.3.4 -port 5555'
    HUB_PORT_4444_TCP_ADDR: 5.6.7.8
    HUB_PORT_4444_TCP_PORT: 4444
```

**Comment by vimagick** [comment]
@ddavison I've tried environment object, the result is the same:

```bash
$ docker-compose up -d
$ docker-compose logs -f
Attaching to selenium_firefox_1
firefox_1  | appending selenium options: -host 1.2.3.4 -port 5555
firefox_1  | Waiting xvfb...
firefox_1  | -bash: 1.2.3.4: command not found
firefox_1  | Waiting xvfb...
firefox_1  | Waiting xvfb...
firefox_1  | Waiting xvfb...
```

The problem is `entry_point.sh` will run this command: 

    SE_OPTS=-host 1.2.3.4 -port 5555 xvfb-run ... java ... -host 1.2.3.4 -port 5555

 (`1.2.3.4` is a command now, the rest are arguments of it).

**Comment by Zechtitus** [comment]
Ran into same thing - Can get the node-chrome working, not the debug variant. This is pretty much required for getting a remote node hooked up to a remote grid.

**Comment by vimagick** [comment]
👍  I'll try it.

**Comment by Zechtitus** [comment]
@vimagick did this work for you? I'm still using my hacked file in the meantime... would be nice to have this fixed in the main project files

**Comment by diemol** [comment]
This was solved with #459, which was merged and released, this issue can be closed.
